modules/osdk-java-implement-controller.adoc

```diff
@@ -0,0 +1,180 @@
+             resource.getStatus().setNodes(podNames);
+             return UpdateControl.updateResource(resource);
+        }
+
+        return UpdateControl.noUpdate();
+    }
+
+    private Map<String, String> labelsForMemcached(Memcached m) {
+        Map<String, String> labels = new HashMap<>();
+        labels.put("app", "memcached");
+        labels.put("memcached_cr", m.getMetadata().getName());
+        return labels;
+    }
+
+    private Deployment createMemcachedDeployment(Memcached m) {
+        return new DeploymentBuilder()
+            .withMetadata(
+                new ObjectMetaBuilder()
+                    .withName(m.getMetadata().getName())
+                    .withNamespace(m.getMetadata().getNamespace())
+                    .withOwnerReferences(
+                        new OwnerReferenceBuilder()
+                            .withApiVersion("v1")
+                            .withKind("Memcached")
+                            .withName(m.getMetadata().getName())
+                            .withUid(m.getMetadata().getUid())
+                            .build())
+                    .build())
+            .withSpec(
+                new DeploymentSpecBuilder()
+                    .withReplicas(m.getSpec().getSize())
+                    .withSelector(
+                        new LabelSelectorBuilder().withMatchLabels(labelsForMemcached(m)).build())
+                    .withTemplate(
+                        new PodTemplateSpecBuilder()
+                            .withMetadata(
+                                new ObjectMetaBuilder().withLabels(labelsForMemcached(m)).build())
+                            .withSpec(
+                                new PodSpecBuilder()
+                                    .withContainers(
+                                        new ContainerBuilder()
+                                            .withImage("memcached:1.4.36-alpine")
+                                            .withName("memcached")
+                                            .withCommand("memcached", "-m=64", "-o", "modern", "-v")
+                                            .withPorts(
+                                                new ContainerPortBuilder()
+                                                    .withContainerPort(11211)
+                                                    .withName("memcached")
+                                                    .build())
+                                            .build())
+                                    .build())
+                            .build())
+                    .build())
+            .build();
+    }
+----
+====
++
+The example controller runs the following reconciliation logic for each `Memcached` custom resource (CR):
++
+--
+* Create a Memcached deployment if it does not exist.
+* Ensure that the deployment size is the same as specified by the `Memcached` CR spec.
```

Comment:
```suggestion
* Ensures that the deployment size matches the size specified by the `Memcached` CR spec.
```
The _same as specified_ bit is throwing me off a little. Though I am not sure I like my rewrite any better. WDYT? Not a blocker if we leave things alone.